src/data-workspace/data-entry-cell/entry-field-input.js

```diff
@@ -16,9 +16,10 @@ import { useDataValueParams } from './use-data-value-params.js'
 import { VALUE_TYPES } from './value-types.js'
 
 function createCurrentItem({ de, coc, dataValueSet }) {
-    if (dataValueSet?.data[de.id]?.[coc.id]) {
+    const dataValue = dataValueSet?.data.dataValues[de.id]?.[coc.id]
```

Comment:
this is the actual fix for date/name not being displayed correctly .. the data exists in `.dataValues`